test/WebpackAssetsManifest-test.js

```diff
@@ -342,6 +342,20 @@ describe('WebpackAssetsManifest', function() {
 
         assert.isTrue(manifest.inDevServer());
       });
+
+      it('Works correct with null prototypes outputFileSystem', function() {
+        const config = configs.hello();
+
+        config.output.path = '/';
+
+        const compiler = makeCompiler(config);
+        compiler.outputFileSystem = Object.create(null);
```

Comment:
Instead of using an empty object, lets just fake the `null prototype` so that we try and match what Jest is doing.

```suggestion

        Object.setPrototypeOf(compiler.outputFileSystem, null);

```